content/applications/productivity/discuss/advanced/email_servers.rst

```diff
@@ -250,4 +264,4 @@ server here selected for mass mails.
 Note that in this case, you have to set your domain's Sender Policy Framework (SPF) records to
 include both transactional and mass mail servers. If your server resides with xxxx.odoo.com, the
 available options are Sendinblue and Mailchimp, as your e-mails would be originated from the
-xxxx.odoo.com domain.
\ No newline at end of file
+xxxx.odoo.com domain.
```

Comment:
In this paragraph can you change xxxx.odoo.com domain to: 

`mycompany.odoo.com domain`

We made this change so that the "xxxx" not be taken out of context.